src/components/plots/stackedAreaPlot.ts

```diff
@@ -75,15 +75,11 @@ export module Plots {
       this._getAnimator("baseline").animate(this._baseline, baselineAttr);
     }
 
-    protected _updateYDomainer() {
-      super._updateYDomainer();
+    protected _updateYScale() {
       var scale = <QuantitativeScale<any>> this.y().scale;
-      if (!scale._userSetDomainer) {
-        scale.domainer().addPaddingException(this, 0)
-                        .addIncludedValue(this, 0);
-        // prepending "AREA_PLOT" is unnecessary but reduces likely of user accidentally creating collisions
-        scale._autoDomainIfAutomaticMode();
-      }
+      scale.addPaddingException(this, 0);
+      scale.addIncludedValue(this, 0);
+      scale._autoDomainIfAutomaticMode();
```

Comment:
Future PR to deal with #2057.

I think `_autoDomainIfAutomaticMode()` is only ever invoked when:
* padding exceptions are added/removed
* included values are added/removed
* an ExtentsProvider is added/removed/updated

Given that there are methods that are invoked whenever the first three happen, we could just call `_autoDomainIfAutomaticMode()` in each of those methods.

Speaking of which, the `_autoDomainIfAutomaticMode()` is redundant here, since that gets invoked when `addPaddingException()` and `addIncludedValue()` are called.